+dj/Relvar.m

```diff
@@ -317,8 +317,8 @@ function insert(self, tuples, command)
                 valueStr = '';
                 for i = find(ix)
                     [v, placeholder] = makePlaceholder(i, tuple.(header.attributes(i).name));
-                    if ~isempty(v) && ( ...
-                            (~isnumeric(v) && ~islogical(v)) || (~isscalar(v) && ~isempty(v)))
+                    if header.attributes(i).isBlob || header.attributes(i).isString && ...
```

Comment:
shouldn't this be
```matlab
if  (header.attributes(i).isBlob || header.attributes(i).isString) && ~isempty(v)
```
(missing parenthesis)

Reply:
@dimitri-yatsenko actually this is not the most updated one. Have a look in #220. That is the one that should be reviewed.